lib/components/top_bar.dart

```diff
@@ -147,7 +147,7 @@ List<StreamSubscription> listeners=[];
       backgroundColor: Styles.primaryBackgroundColor,
       padding: EdgeInsets.symmetric(horizontal: 10, vertical: 6),
       onPressed: () {
-        if(isConnected){
+        if(!isConnected){
```

Comment:
why this change?

Reply:
ah yes , removed it , was only for development